src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+
+  <section className="Person">
+    <>
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      <p className="Person__age">
+        {person.age ? `I am ${person.age}` : person.age === 0}
+      </p>
+      <p className="Person__partner">
+        {person.isMarried ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}` : 'I am not married'}
```

Comment:
Nested ternary looks too complicated, create a separate variable for person partner and use it here:
`const partner = person.sex === 'f' ? 'husband' : 'wife'`

Reply:
![image](https://github.com/mate-academy/react_person/assets/143103697/290ad689-0917-4861-9bcb-7f86d5a066e7)
I get an error when creating a variable